app/api/helpers/scheduled_jobs.py

```diff
@@ -58,144 +67,192 @@ def send_after_event_mail():
-                                                            incomplete_invoice.amount,
-                                                            app_name,
-                                                            link)
-                send_followup_notif_monthly_fee_payment(incomplete_invoice.user,
-                                                        incomplete_invoice.event.name,
-                                                        prev_month,
-                                                        incomplete_invoice.amount,
-                                                        app_name,
-                                                        link,
-                                                        incomplete_invoice.event.id)
+                link = '{}/event-invoice/{}/review'.format(
+                    frontend_url, incomplete_invoice.identifier
+                )
+                send_followup_email_for_monthly_fee_payment(
+                    incomplete_invoice.user.email,
+                    incomplete_invoice.event.name,
+                    prev_month,
+                    incomplete_invoice.amount,
+                    app_name,
+                    link,
+                )
+                send_followup_notif_monthly_fee_payment(
+                    incomplete_invoice.user,
+                    incomplete_invoice.event.name,
+                    prev_month,
+                    incomplete_invoice.amount,
+                    app_name,
+                    link,
+                    incomplete_invoice.event.id,
+                )
 
 
 @celery.task(base=RequestContextTask, name='expire.pending.tickets')
 def expire_pending_tickets():
     from app.instance import current_app as app
+
     with app.app_context():
-        db.session.query(Order).filter(Order.status == 'pending',
-                                       (Order.created_at + datetime.timedelta(minutes=30)) <= datetime.datetime.now()).\
-                                       update({'status': 'expired'})
+        db.session.query(Order).filter(
+            Order.status == 'pending',
+            (Order.created_at + datetime.timedelta(minutes=30))
+            <= datetime.datetime.now(),
+        ).update({'status': 'expired'})
         db.session.commit()
 
 
 @celery.task(base=RequestContextTask, name='delete.ticket.holders.no.order.id')
 def delete_ticket_holders_no_order_id():
     from app.instance import current_app as app
+
     with app.app_context():
         order_expiry_time = get_settings()['order_expiry_time']
-        TicketHolder.query.filter(TicketHolder.order_id == None, TicketHolder.deleted_at.is_(None),
-                                  TicketHolder.created_at + datetime.timedelta(minutes=order_expiry_time)
-                                  < datetime.datetime.utcnow()).delete(synchronize_session=False)
+        TicketHolder.query.filter(
+            TicketHolder.order_id == None,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [comparison to None should be 'if cond is None:'](https://app.codacy.com/manual/fossasia/open-event-orga-server/pullRequest?prid=4884175)